Source/Examples/ExampleLibrary/Utilities/PlotModelUtilities.cs

```diff
@@ -55,7 +55,6 @@ public static class PlotModelUtilities
         /// </summary>
         private static readonly HashSet<Type> NonTransposableDataSpaceAnnotationTypes = new HashSet<Type>
         {
-            typeof(TileMapAnnotation),
```

Comment:
They aren't transposable, because we can't draw rotated images.

Reply:
Yes, but `TileMapAnnotation` doesn't implement `ITransposablePlotElement`, therefore it should not even be considered to be transposable. But now that you mention this I do realize this check is never actually made in `PlotModelUtilities`. I will get on that.